src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <section class="container">
+      <div class="container__card card" data-qa="card">
+        <img
+          class="image card__image"
+          src="./images/imac.jpeg"
+          alt="apple-imac-photo"
+        >
+
+        <div class="title card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </div>
+
+        <div class="product-code card__product-code">Product code: 195434</div>
+
+        <div class="card__rating rating">
+          <div class="rating__stars">
+            <div class="stars stars--4">
```

Comment:
p. 6 of checklist.
Remove <div class="card__rating rating".
Adapt such structure to your code: 
`<div class="stars card__stars">
        <div class="stars__quantity stars__quantity--4">
          <div class="stars__star"></div>`